src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,12 @@
 package mate.academy.service;
 
 public class AuthenticationService {
-    /**
-     * Imagine that some user wants to login to your site.
-     * You should check if user credentials (login and password) are valid or not.
-     * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
-     * @param password - user's password
-     * @return true if user by email exists and passed password is equal to user's password.
-     * Return false in any other cases.
-     */
+
     public boolean login(String email, String password) {
+        UserService user = new UserService();
+        if (user.findByEmail(email) != null) {
```

Comment:
store result of this call, don't call this method twice
return one boolean expression